src/style.css

```diff
@@ -1,3 +1,78 @@
+/* #region aproved */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
Add fallback font

Reply:
"if roboto is absent, use any other sens-serif font", whats wrong with this code? When we learn how to add fallback fonts, in your platform, it were a mistake when I add Arial as second font. But Ok